admin/site-admin/people.md

```diff
@@ -0,0 +1,7 @@
+# View and manage users
+
+Access **People** from the Stoplight dashboard by selecting:
+
+**admin** > **Site Admin** > **People**
+
+You can view a list of users and search for an individual user.
```

Comment:
You can all remove a persons record completely from the system and change their access level (if you are a site-admin, you can promote others to site-admin)

Reply:
I think this functionality was added this sprint? This is why I'd suggest documenting a sprint behind (sprint + 1). Will add this now, but normally catch this sort of thing in the next sprint.